connector_importer_source_sftp/tests/common.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
+# @author: Simone Orsi <simahawk@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.addons.component.tests.common import (
+    SavepointComponentCase,
+    SavepointComponentRegistryCase,
+)
+from odoo.addons.connector_importer.tests.common import TestImporterMixin
+
+
+class TestSourceCSVSFTPMixin(object):
```

Comment:
```suggestion
class TestSourceCSVSFTPMixin():
```

Reply:
but I like it so much.... :stuck_out_tongue_closed_eyes: